.github/actions/synthetics-stack-e2e/action.yml

```diff
@@ -0,0 +1,136 @@
+---
+name: synthetics-stack-e2e
+description: |
+  An action that orchestrates the Synthetics Stack E2E tests for a Kibana PR.
+  It first creates a Kibana docker image, then it deploys an ESS cluster with the created docker image,
+  and finally it runs the Synthetics Stack E2E tests against the deployed ESS cluster.
+
+inputs:
+  buildkite-pipeline-slug:
+    description: 'The Buildkite pipeline slug'
+    default: kibana-pr-synthetics-stack-e2e-ci
+    required: false
+  vault-url:
+    description: 'Vault URL'
+    required: true
+  vault-role-id:
+    description: 'Vault role ID'
+    required: true
+  vault-secret-id:
+    description: 'Vault secret ID'
+    required: true
+  comment-id:
+    description: 'The GitHub Comment ID'
+    default: ${{ github.event.comment.id }}
+  commit-sha:
+    description: 'The commit SHA'
+    default: ${{ github.event.pull_request.head.sha }}
+    required: false
+  pr-number:
+    description: 'The PR number'
+    default: ${{ github.event.issue.number }}
+    required: false
+  user:
+    description: 'The GitHub user that triggered the workflow'
+    default: ${{ github.triggering_actor }}
+    required: true
+  context:
+    description: 'The commit status context'
+    default: synthetics-stack-e2e
+    required: false
+  comment-command:
+    description: 'The comment command'
+    default: '/synthetics-stack-e2e'
+    required: false
+
+runs:
+  using: composite
+  steps:
+    # If the action is used in a pull_request[_target] workflow
+    - if: github.event.pull_request
+      name: Create initial failing commit status
```

Comment:
I'm not totally onboarded with this particular behaviour. https://github.com/elastic/kibana/pull/165993#discussion_r1333119046 was my last comment.

Reply:
with the new structure, the commit status context will only be created if the PR is from an elastic member.

otherwise, no additional commit status context will be created and the workflow status check will just be green.